### ISSUE TITLE
Re-write in Rust? Mentor needed 🦀 

### ISSUE THREAD

**Issue by cuducos** [cuducos/createnv#3]
I think this sort of tool is ideal to be written in Rust: way faster, type check security by default, we can create a proper compiler (instead of trusting regexes).

At the same time, I'm just a beginner in Rus and I guess this would be a great project to advance in my Rust studies. 

The point is, I wouldn't like to do it alone as I feel like it would be too time-consuming and full of roadblocks a newbie would take longer than needed to understand. We can do all async in GitHub, with quality code reviews.

**TLDR**

If you find this project interesting and think you can help me in re-writing it in Rust, I'd love to have you as a mentor — and if you want to do this as a pro, I can compensate you for your time (surely, depending on the fee!).

**Comment by cuducos** [comment]
Mentorship in progress : )
